registry/client.go

```diff
@@ -54,7 +54,7 @@ var (
 
 func init() {
 	configureDiscoveryRetry()
-	configureRequestTimeout()
+	ConfigureProviderRequestTimeout()
```

Comment:
Same comment here.

Reply:
oh no, thank you so much for catching these! I caught a few earlier find&replace errors but my eyeballs went blind to the diff after awhile.